pkg/workload/schemachange/operation_generator.go

```diff
@@ -326,6 +326,31 @@ var opDeclarative = []bool{
 	validate:                false,
 }
 
+var opDeclarativeVersion = []clusterversion.Key{
+	addColumn:               clusterversion.V22_2,
```

Comment:
I was debating if we should use the check if supported function from the builder, but the problem is mixed version states here. The binary is built against a new version for the test, I'll add a comment with a rationale.

// This workload will maintain its own list of supported versions for declarative
// schema changer, since the cluster we are running against can be downlevel. 
// The declarative schema changer builder does have a supported list but it's not
// sufficient for that reason.